Cannot debug 'Unit Tests' from vscode's launch.json

Issue Type: <b>Bug</b>

**Repo**
1. In VS Code code base
1. Try launch the `Unit tests` debug configuration

**Bug**
In the debug console, see the following error:

```
The factory method of "vs/platform/node/product" has thrown an exception
Error: [UriError]: Scheme is missing: {scheme: "", authority: "", path: "/Users/matb/projects/vscode/src/", query: "", fragment: ""}
    at _validateUri (/Users/matb/projects/vscode/out/vs/base/common/uri.js:39:23)
    at _URI.URI (/Users/matb/projects/vscode/out/vs/base/common/uri.js:126:17)
    at new _URI (/Users/matb/projects/vscode/out/vs/base/common/uri.js:324:51)
    at Function.URI.parse (/Users/matb/projects/vscode/out/vs/base/common/uri.js:235:20)
    at Object.getPathFromAmdModule (/Users/matb/projects/vscode/out/vs/base/common/amd.js:9:26)
    at Object.<anonymous> (/Users/matb/projects/vscode/out/vs/platform/node/product.js:8:39)
    at Function.Module._safeInvokeFunction (/Users/matb/projects/vscode/out/vs/loader.js:951:45)
    at Function.Module._invokeFactory (/Users/matb/projects/vscode/out/vs/loader.js:970:29)
    at Module.complete (/Users/matb/projects/vscode/out/vs/loader.js:983:36)
    at ModuleManager._onModuleComplete (/Users/matb/projects/vscode/out/vs/loader.js:1583:20)
    at ModuleManager._resolve (/Users/matb/projects/vscode/out/vs/loader.js:1545:22)
    at ModuleManager.defineModule (/Users/matb/projects/vscode/out/vs/loader.js:1209:18)
    at ModuleManager._onLoad (/Users/matb/projects/vscode/out/vs/loader.js:1286:22)
    at Object.callback (/Users/matb/projects/vscode/out/vs/loader.js:1456:31)
    at OnlyOnceScriptLoader.triggerCallback (/Users/matb/projects/vscode/out/vs/loader.js:588:36)
    at /Users/matb/projects/vscode/out/vs/loader.js:582:90
    at NodeScriptLoader._loadAndEvalScript (/Users/matb/projects/vscode/out/vs/loader.js:814:17)
    at /Users/matb/projects/vscode/out/vs/loader.js:780:31
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:511:3)
```

I've tried a clean `yarn` and rebuild but haven't been able to get this working

VS Code version: Code - Insiders 1.30.0-insider (ebe9ea85a0a0bb087dccc98949542605dfc51f6f, 2018-11-19T19:59:44.607Z)
OS version: Darwin x64 18.2.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz (8 x 2200)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 3|
|Memory (System)|16.00GB (0.12GB free)|
|Process Argv|-psn_0_15298198|
|Screen Reader|no|
|VM|0%|

</details>
<!-- generated by issue reporter -->

After disabling the uri check temporarily, I now get:

```
{ errorCode: 'load',
  moduleId: 'electron',
  neededBy: 
   [ 'vs/workbench/parts/extensions/electron-browser/extensionsActions' ],
  detail: { Error: Cannot find module 'electron'
    at Function.Module._resolveFilename (module.js:543:15)
    at Function.Module._load (module.js:473:25)
    at Module.require (module.js:586:17)
    at Module.patchedRequire [as require] (/Users/matb/projects/vscode/node_modules/diagnostic-channel/dist/src/patchRequire.js:14:46)
    at require (internal/module.js:11:18)
    at NodeScriptLoader.load (/Users/matb/projects/vscode/out/vs/loader.js:741:39)
    at OnlyOnceScriptLoader.load (/Users/matb/projects/vscode/out/vs/loader.js:582:32)
    at loadNextPath (/Users/matb/projects/vscode/out/vs/loader.js:1451:41)
    at Object.errorback (/Users/matb/projects/vscode/out/vs/loader.js:1459:25)
    at OnlyOnceScriptLoader.triggerErrorback (/Users/matb/projects/vscode/out/vs/loader.js:595:36)
    at /Users/matb/projects/vscode/out/vs/loader.js:582:151
    at ReadFileContext.callback (/Users/matb/projects/vscode/out/vs/loader.js:754:25)
    at FSReqWrap.readFileAfterOpen [as oncomplete] (fs.js:420:13) code: 'MODULE_NOT_FOUND' } }
Waiting for the debugger to disconnect...
```

--- a/.vscode/launch.json
+++ b/.vscode/launch.json
@@ -194,8 +194,8 @@
 		{
 			"type": "node",
 			"request": "launch",
-			"name": "Unit Tests",
-			"program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
+			"name": "Run Unit Tests",
+			"program": "${workspaceFolder}/test/electron/index.js",
 			"runtimeExecutable": "${workspaceFolder}/.build/electron/Code - OSS.app/Contents/MacOS/Electron",
 			"windows": {
 				"runtimeExecutable": "${workspaceFolder}/.build/electron/Code - OSS.exe"
@@ -205,14 +205,9 @@
 			},
 			"outputCapture": "std",
 			"args": [
-				"--delay",
-				"--timeout",
-				"2000"
+				"--remote-debugging-port=9222"
 			],
 			"cwd": "${workspaceFolder}",
-			"env": {
-				"ELECTRON_RUN_AS_NODE": "true"
-			},
 			"outFiles": [
 				"${workspaceFolder}/out/**/*.js"
 			]
@@ -269,6 +264,13 @@
 				"Launch VS Code",
 				"Attach to Extension Host"
 			]
-		}
+		},
+		{
+			"name": "Debug Unit Tests",
+			"configurations": [
+				"Attach to VS Code",
+				"Run Unit Tests"
+			]
+		},
 	]
 }

